sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -129,8 +129,16 @@ def _populate_message_properties(self, message):
 
     def _enhanced_message_received(self, frame, message):
         # pylint: disable=protected-access
-        self._handler._was_message_received = True
+        # self._handler._was_message_received = True
```

Comment:
If this is no longer needed in the pyamqp client (it used to be used for batch optimization I think, do we do that differently now?), you can remove it. Looks like it snuck into the async method below as well.